server/service/apple_mdm.go

```
@@ -553,9 +553,15 @@ func validateConfigProfileFleetVariables(appConfig *fleet.AppConfig, contents st
 			return nil, err
 		}
 	}
-	// TODO: Both Custom SCEP and NDES share RenewalID -- clean up this validation for corner cases such as: both SCEP and NDES variables present in
-	//  one profile
-	if customSCEPVars.Found() && (!ndesVars.Found() || ndesVars.RenewalOnly()) {
+	// Since both custom SCEP and NDES share the renewal ID Fleet variable, we need to figure out which one to validate.
+	if customSCEPVars.Found() && ndesVars.Found() {
```

Do we want to throw an error if they only specified the renewal ID var?

---

Yes, we do throw an error. It is part of Figma and I added a test.